atomisticparsers/gromacs/parser.py

```diff
@@ -93,7 +93,6 @@ def str_to_energies(val_in):
                 n_chars_val = len(n_chars_val.group(1)) if n_chars_val is not None else None
```

Comment:
we can get rid of the for loop if we do re.search(rf'( +{"|".join(thermo_common)})', ' val_in)